content/en/community/community-meeting.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Monthly Community Meeting
```

Comment:
The community meeting, office-hours and, moc docs are external content and pulled in externally through the [hack/gen-content.sh](https://github.com/kubernetes/contributor-site/blob/master/hack/gen-content.sh) script.

To permanently remove them, they will need to be modified here:

https://github.com/kubernetes/contributor-site/blob/5500385f6d00cf7682b4aa8ece81c089595e418a/external-sources/kubernetes/community#L17-L19

We want to retain the community meeting, its new location should be under community. The other 2 (office-horus/moc) can be removed. They aren't something we're actively doing these days. 👍 

We will also need to remove the `.gitignore` file in the events dir, its no longer adding anything:
https://github.com/kubernetes/contributor-site/blob/master/content/en/events/.gitignore

and add community-meeting.md to the .gitignore in the community dir
https://github.com/kubernetes/contributor-site/blob/master/content/en/community/.gitignore


As an aside, a follow up PR should be done to the community meeting docs in k/community to change its weight to 3 to position it better under the community section.

Reply:
whats the role of .gitignore here? for eg does `.gitignore` excludes `community-meeting.md` from something while building the website?